src/utils.js

```diff
@@ -612,3 +627,21 @@ export const getDecodedParams = (params) =>
     }
     return acc
   }, {})
+
+export const getClassNames = (styles, classes) => {
```

Comment:
as this function is only used in templates, perhaps we should consider shifting it to `templates/utils/<some_filename>.ts` (and migrate it to ts)

Reply:
also, if `styles` is going to be `editorStyles` most of the time, we should set it as a default argument so callers can forget about it. this does mean that we need to document (as a JSDoc) when the `styles` might change so that callers are cognizant when using this function